ethereum/viper
# Refactor parser parse_stmt method

### - What I did
I broke the `parse_stmt` method out into it's own class.

### - How I did it
1) Turned the `parse_stmt` conditional into a dictionary
2) Move `parser.py` into a parser folder
3) Create a seperate `stmt.py` file

### - How to verify it
Look through my PR to make sure my refactors increase simplicity and don't change any functionality.

### - Description for the changelog
None
### - Cute Animal Picture
![image](https://user-images.githubusercontent.com/17552858/31060824-3922c798-a6d7-11e7-9675-1ee733336709.png)
